light-curve/light_curve/light_curve_py/features/rainbow/symmetric.py

```diff
@@ -0,0 +1,220 @@
+            amplitude, rise_time, fall_time = params[self.p.bol_idx]
+            amplitude = m_scaler.undo_scale(amplitude)
+            rise_time = t_scaler.undo_scale(rise_time)
+            fall_time = t_scaler.undo_scale(fall_time)
+            params[self.p.bol_idx] = amplitude, rise_time, fall_time
+
+        if self.with_temperature_evolution:
+            T_min, T_max, k_sig = params[self.p.temp_idx]
+            k_sig = t_scaler.undo_scale(k_sig)
+            params[self.p.temp_idx] = T_min, T_max, k_sig
+
+    def _initial_guesses(self, t, m, band) -> Dict[str, float]:
+        if self.with_rise_only:
+            t_rise = 1.0
+        else:
+            t_rise = 0.1
+        t_fall = 0.1
+
+        # The amplitude here does not actually correspond to the amplitude of m values!
+        if self.with_baseline:
+            A = np.ptp(m)
+        else:
+            A = np.max(m)
+
+        params = {}
+
+        # Why do we have to strictly follow the order of parameters here?..
+        params["reference_time"] = t[np.argmax(m)]
+        params["amplitude"] = A
+        params["rise_time"] = t_rise
+
+        if not self.with_rise_only:
+            params["fall_time"] = t_fall
+
+        params["Tmin"] = 7000.0
+
+        if self.with_temperature_evolution:
+            params["Tmax"] = 10000.0
+            params["k_sig"] = 1.0
+
+        return params
+
+    def _limits(self, t, m, band) -> Dict[str, Tuple[float, float]]:
+        t_amplitude = np.ptp(t)
+        if self.with_baseline:
+            m_amplitude = np.ptp(m)
+        else:
+            m_amplitude = np.max(m)
+
+        limits = {}
+
+        # Why do we have to strictly follow the order of parameters here?..
+        limits["reference_time"] = (np.min(t) - 10 * t_amplitude, np.max(t) + 10 * t_amplitude)
+        limits["amplitude"] = (0.0, 10 * m_amplitude)
+        limits["rise_time"] = (1e-4, 10 * t_amplitude)
+
+        if not self.with_rise_only:
+            limits["fall_time"] = (1e-4, 10 * t_amplitude)
+
+        limits["Tmin"] = (1e2, 1e6)  # K
+
+        if self.with_temperature_evolution:
+            limits["Tmax"] = (1e2, 1e6)  # K
+            limits["k_sig"] = (1e-4, 10.0 * t_amplitude)
+
+        return limits
+
+    def _baseline_initial_guesses(self, t, m, band) -> Dict[str, float]:
+        """Initial guesses for the baseline parameters."""
+        return {self.p.baseline_parameter_name(b): np.median(m[band == b]) for b in self.bands.names}
+
+    def peak_time(self, params) -> float:
+        """Returns true bolometric peak position for given parameters"""
+        if self.with_rise_only:
+            t0, amplitude, rise_time = params[self.p.all_bol_idx]
+
+            # It is not, strictly speaking, defined for rising only
+            return t0
+        else:
+            t0, amplitude, rise_time, fall_time = params[self.p.all_bol_idx]
+
+            return t0 + np.log(fall_time / rise_time) * rise_time * fall_time / (rise_time + fall_time)
```

Comment:
We don't need `else` here

```suggestion
        t0, amplitude, rise_time, fall_time = params[self.p.all_bol_idx]
        return t0 + np.log(fall_time / rise_time) * rise_time * fall_time / (rise_time + fall_time)
```